tests/components/admin_console/manage_teams_modal/manage_teams_modal.test.jsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ManageTeamsModal from 'components/admin_console/manage_teams_modal/manage_teams_modal.jsx';
+
+describe('ManageTeamsModal', () => {
+    function emptyFunc() {} // eslint-disable-line no-empty-function
+
+    const baseProps = {
+        onModalDismissed: emptyFunc,
+        show: true,
+        user: {
+            id: 'currentUserId',
+            last_picture_update: '1234',
+            email: 'currentUser@test.com',
+            roles: 'system_user',
+            username: 'currentUsername',
+        },
+        updateTeamMemberSchemeRoles: emptyFunc,
+        getTeamMembersForUser: emptyFunc,
```

Comment:
I only found out about this when reviewing one of Elias's PRs, but you set each of these to `jest.fn` once without redefining the actions like you do in "should match snapshot init". For example, this is valid and passes
```javascript
describe('component', () => {
    const action = jest.fn();

    test('action called', () => {
        action();
        expect(action).toHaveBeenCalled();
    });

    test('action not called', () => {
        expect(action).not.toHaveBeenCalled();
    });
});
```

Reply:
Done